examples/testmonitor/README.md

```diff
@@ -28,4 +28,5 @@ Test Monitor Examples
 ------------
 
 - [Results](results): Demonstrates how to use the SystemLink Test Monitor API to create test results and steps.
+- [File Attachment](fileattachment): Demonstrates how to use the SystemLink Test Monitor API to attach files that are uploaded to the SystemLink server using the SystemLink File API.
```

Comment:
Attach them to a test result, right? Maybe

> to attach files from the SystemLink server to a test result using the SystemLink File API.

Also curious how attached we are to specifying "SystemLink" before API names. I realize it might be more necessary in a non-NI-specific space.